FoldingTabBar/View/CustomHeightTabBar/YALCustomHeightTabBar.h

```diff
@@ -0,0 +1,9 @@
+// For License please refer to LICENSE file in the root of YALAnimatingTabBarController project
+
+#import <UIKit/UIKit.h>
+
+@interface YALCustomHeightTabBar : UITabBar
+
+@property (nonatomic) CGFloat barHeight;
```

Comment:
Configure property 'barHeight' using 'tabBarViewHeight' property of YALFoldingTabBarController. Users configure height of tab bar when they set tabBarViewHeight property, so we should use it. Thanks.